staging/src/k8s.io/kubectl/pkg/cmd/diff/diff.go

```diff
@@ -257,6 +257,7 @@ type InfoObject struct {
 	Force           bool
 	ServerSideApply bool
 	ForceConflicts  bool
+	genericclioptions.IOStreams
```

Comment:
This class has way too many fields that dont' really make sense there. This is a fine fix, but we should consider refactoring this a little bit.